refinery/user_files_manager/tests.py

```diff
@@ -8,11 +8,13 @@
 from django.http import QueryDict
 from django.test import RequestFactory, TestCase, override_settings
 
+from constants import REFINERY_SOLR_DOC_LIMIT
```

Comment:
`import constants` and move it into the "local" import section below since this is not a 3rd party module